cmd/context/utils.go

```diff
@@ -186,12 +186,43 @@ func LoadStackWithContext(ctx context.Context, name, namespace string, stackPath
 	return s, nil
 }
 
+//LoadManifestV2WithContext initializes the okteto context taking into account command flags and manifest namespace/context fields
+func LoadManifestV2WithContext(ctx context.Context, namespace, path string) error {
+	ctxOptions := &ContextOptions{
+		Namespace: namespace,
+		Show:      true,
+	}
+
+	cwd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	manifest, err := GetManifestV2(cwd, path)
+	if err != nil {
+		if err != errors.ErrManifestNotFound {
```

Comment:
Is this still able to run if the manifest is not found? If so, I think this is worth a comment as to why the error handling is this way.

Reply:
Added a comment. I also open an issue to refactor how the manifest is read at the moment. Right now the logic is split into different functions and it should be a single function, with a cleaner interface, and in a different package:
https://github.com/okteto/okteto/issues/2111